src/Illuminate/Mail/Mailable.php

```diff
@@ -164,6 +164,20 @@ class Mailable implements MailableContract, Renderable
      */
     protected $assertionableRenderStrings;
 
+    /**
+     * The metadata for the message.
+     *
+     * @var array
+     */
+    protected $metadata = [];
+
+    /**
+     * The tag for the message.
+     *
+     * @var string
+     */
+    protected $tag;
```

Comment:
This isn't correct. You should be able to add multiple tags.

Reply:
Corrected.